src/corebluetooth.cs

```diff
@@ -763,21 +763,25 @@ interface CBUUID : NSCopying {
 		[Field ("CBUUIDCharacteristicAggregateFormatString")]
 		NSString CharacteristicAggregateFormatString { get; }
 
-#if MONOMAC
 		[Internal]
 		[Field ("CBUUIDValidRangeString")]
 		[Introduced (PlatformName.MacOSX, 10, 12)]
 		[Deprecated (PlatformName.MacOSX, 10, 13)]
 		[Obsoleted (PlatformName.MacOSX, 10, 13)]
+		[NoiOS][NoTV][NoWatch][NoMacCatalyst]
 		NSString CBUUIDValidRangeString { get; }
 
+#if MONOMAC && !NET
 		[Internal]
 		[Mac (10,13)]
 		[Field ("CBUUIDCharacteristicValidRangeString")]
 		NSString CBUUIDCharacteristicValidRangeString { get; }
 #else
 		[iOS (10,0)]
 		[TV (10,0)]
+		[Watch (4,0)]
+		[Mac (10,13)]
+		[MacCatalyst (13,0)]
```

Comment:
Also it looks like the min MacCatalyst version we support is 13.1, I don't think it should matter too much though right?